components/locid/src/helpers.rs

```diff
@@ -13,7 +14,7 @@ use litemap::store::*;
 #[derive(Debug, Clone, PartialEq, Eq, Hash, PartialOrd, Ord)]
 pub(crate) enum ShortVec<T> {
```

Comment:
This should maybe be called `ShortSlice` now?

Reply:
Renamed it to `ShortSlice`. We still have `insert` and `remove` which "might" reallocate, but that was sort of true for vector too.